test/test_pkgconfig.rb

```diff
@@ -168,12 +168,27 @@ def test_requires
     end
 
     def test_recursive_requires
-        pkg = Utilrb::PkgConfig.get('test_pkgconfig_recursive_require_loop_a')
+        pkg = Utilrb::PkgConfig.parse_dependencies('test_pkgconfig_recursive_require_loop_a')[0]
+        
         assert_equal ['test_pkgconfig_recursive_require_loop_b', 'test_pkgconfig_recursive_require_loop_c'],
             pkg.requires.map(&:name)
 
-        pkg = Utilrb::PkgConfig.get('test_pkgconfig_recursive_require_loop_b')
-        assert_equal ['test_pkgconfig_recursive_require_loop_a', 'test_pkgconfig_recursive_require_loop_d'],
-            pkg.requires.map(&:name)        
+        assert_equal ['test_pkgconfig_recursive_require_loop_a', 'test_pkgconfig_recursive_require_loop_b', 'test_pkgconfig_recursive_require_loop_d'],
+            pkg.requires[0].requires.map(&:name)
+
+
+        pkg = Utilrb::PkgConfig.parse_dependencies('test_pkgconfig_recursive_require_loop_b')[0]
+        assert_equal ['test_pkgconfig_recursive_require_loop_a', 'test_pkgconfig_recursive_require_loop_b', 'test_pkgconfig_recursive_require_loop_d'],
+            pkg.requires.map(&:name)
+            
+        assert_equal ['test_pkgconfig_recursive_require_loop_b', 'test_pkgconfig_recursive_require_loop_c'],
+            pkg.requires[0].requires.map(&:name)
+    
+    end
+
+    def test_version_not_number
```

Comment:
There's one edge-case that I think you are already covering, but I don't know for sure. Best would be to write a test for it: `A >= B >= 0.2`